contact.html

```diff
@@ -53,5 +53,7 @@ <h1>Scan the code or hit the button</h1>
     >
 
     <script src="js/mobile-toggle.js" async defer></script>
+    <footer-ss></footer-ss>
```

Comment:
@typhlosion64 would appreciate your assistance looking into the styling of the footer on this page